oeps/architectural-decisions/oep-0065-frontend-composability.rst

```diff
@@ -0,0 +1,492 @@
+OEP-65: Frontend Composability
+##############################
+
+.. list-table::
+   :widths: 25 75
+
+   * - OEP
+     - :doc:`OEP-65 <oep-0065-arch-frontend-composability>`
```

Comment:
Note: This is the preferred way of titling the doc (`oep-####-(type)-(name)`) but the actual title of this file doesn't have the `arch` in it

Reply:
I wonder if we lose all the comments if I rename the doc... 😬

Reply:
Yeah, that is actually possible. Maybe get it perfect on this PR, close the PR, and open a new one with the properly titled document - no discussion, just straight to merge?

Reply:
Yeah, fixing it in a follow-up step sounds like a safer option.